wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/AuroraHostListProvider.java

```diff
@@ -80,9 +80,10 @@ public class AuroraHostListProvider implements DynamicHostListProvider {
           + "ORDER BY LAST_UPDATE_TIMESTAMP";
 
   static final String MYSQL_RETRIEVE_TOPOLOGY_SQL =
-      "SELECT SERVER_ID, SESSION_ID, LAST_UPDATE_TIMESTAMP, REPLICA_LAG_IN_MILLISECONDS "
+      "SELECT SERVER_ID, SESSION_ID"
           + "FROM information_schema.replica_host_status "
-          + "WHERE time_to_sec(timediff(now(), LAST_UPDATE_TIMESTAMP)) <= 300 " // 5 min
+          // filter out nodes that haven't been updated in the last 5 minutes
+          + "WHERE time_to_sec(timediff(now(), LAST_UPDATE_TIMESTAMP)) <= 300 OR SESSION_ID = 'MASTER_SESSION_ID'"
```

Comment:
We should leave REPLICA_LAG_IN_MILLISECONDS in and add it to the postgres version. We should also be ordering by replica_lag_in_milliseconds so that the most recent replica is first

Reply:
We are not using `REPLICA_LAG_IN_MILLISECONDS` in the code right now, so it should be safe to keep it out until we need it. During failover we randomize the readers so this ordering isn't really used.

Reply:
we should not be randomizing readers. We should be using the reader that is the most consistent with the writer.